### ISSUE TITLE
Python stacktrace is not returned by the Python magic

### ISSUE THREAD

**Issue by patrick-nicholson** [maxpoint/spylon-kernel#20]
Using the `%%python` magic when running a `spylon-kernel` notebook in Scala mode, any Python stacktrace is not returned when Python raises an error. Example:

In:
```{python}
%%python
1 / 0
```

Out:
```{scala}
  File ".../lib/python3.5/site-packages/metakernel/magics/python_magic.py", line 25, in exec_code
    exec(code, env)

  File "<string>", line 1, in <module>
```

**Comment by parente** [comment]
Need to look into whether using `eval` has the same net effect, but with the stack trace.

**Comment by parente** [comment]
Tracking upstream where the `%%python` magic is implemented: Calysto/metakernel#147
